pkg/varlinkapi/transfers.go

```diff
@@ -30,10 +30,10 @@ func (i *LibpodAPI) SendFile(call iopodman.VarlinkCall, ftype string, length int
 	}
 
 	// FIXME return parameter
-	if err = call.ReplySendFile("FIXME_file_handle"); err != nil {
-		// If an error occurs while sending the reply, return the error
-		return err
-	}
+	// if err = call.ReplySendFile("FIXME_file_handle"); err != nil {
```

Comment:
This led to issue #3494  without comment. Should I use method SendFileOverVarlink?